contracts/axelar-gateway/src/auth.rs

```diff
@@ -1,16 +1,16 @@
 use axelar_soroban_interfaces::types::{ProofSignature, ProofSigner, WeightedSigner};
-use soroban_sdk::xdr::ToXdr;
-use soroban_sdk::{crypto::Hash, panic_with_error, Bytes, BytesN, Env, Vec};
+use soroban_sdk::{crypto::Hash, panic_with_error, xdr::ToXdr, Bytes, BytesN, Env, Vec};
 
 use crate::error::AuthError;
+use crate::event;
 use crate::storage_types::DataKey;
 use axelar_soroban_interfaces::types::{Proof, WeightedSigners};
 
 pub fn initialize_auth(
     env: Env,
     domain_separator: BytesN<32>,
     minimum_rotation_delay: u64,
-    previous_signer_retention: u64,
+    previous_signers_retention: u64,
     initial_signers: Vec<WeightedSigners>,
 ) {
     if env.storage().instance().has(&DataKey::AuthInitialized) {
```

Comment:
also auth initialize isn't needed anymore since gateway itself should be initialized and it's the same contract